### DISCUSSION TITLE
Show daily reading status

### DISCUSSION THREAD

**Discussion by nitaicharan** [dailydotdev/daily#683] [Feature Requests]
## Request
I would like to have a way to see if in my 'Weekly reading goal' status if I've read already in the current day.

## Reason
It will help the readers, who are interested to keep a reading habit, to track if they have read or not in the current day.

**Comment by idoshamun** [comment]
Hi @nitaicharan,

Thanks for the suggestion. We are actually going to change the mechanism altogether but it will take some time. Stay tuned!
